### ISSUE TITLE
NTR: Aspartoacylase low in cultured skin fibroblasts (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9067]
New term request Aspartoacylase low in cultured skin fibroblasts (segal_031030232035):   UMLS: 1848993

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000354
- label: Reduced aspartoacylase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:8252036
- def: Acitivity of aspartoacylase below the lower limit of normal in cultured fibroblasts. Aspartoacylase (EC 3.5.1.15), also called aminoacylase-2, is an enzyme that hydrolyzes N-acetyl-L-aspartic acid (NAA) to aspartate and acetate.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9067
- synonyms:
